### ISSUE TITLE
[BUG] Neg + smaller pos = neg

### ISSUE THREAD

**Issue by sporevasion** [Ivy-Apps/ivy-wallet#2518]
### Please confirm the following

- [X] I checked [the current issues](https://github.com/ILIYANGERMANOV/ivy-wallet/issues) for duplicate problems

### Describe the bug

Basic math from negative balance after adding income isn't working for me. 

### To Reproduce

1. Start with a running account (months) with a balance of -$10.51. So a deficit.
2. Tap Add Income button to make an adjustment.
3. Enter the amount $9.57.
4. Expected result: -$. 94.  (yes, a negative number)
5. Actual result: $. 94.

Basically: 
(-10.51) + 9.57 = (-.94)

Instead, what the app gave me instead was:
(-10.51) + 9.57 = .94

### Expected behavior

Negative plus a smaller positive should still be negative. 

### Screenshots

. 

### Smartphone

Samsung A21, Android 12

### Additional context

.

**Comment by ILIYANGERMANOV** [comment]
Hmm, good catch! Can you please provide a screen recording so we can reproduce it?

**Comment by sporevasion** [comment]
I left pretty detailed steps to reproduce.
I started to record, but realized I cannot. Since I'm maintaining my real finances, if I delete the entry in question, it will change the balance to an incorrect one. It's honestly more work to keep trying to add expenses and income, both positive and negative, to reproduce exact numbers in the issue, in order to re-capture the steps. Coming from development, I understand how it would be nice, however the steps are precise. A developer should have a sandbox version/dB to tweak the main balance, and perform the single step of adding income.
Sorry, I'm trying to use the app seriously as a replacement. 🤷‍♀️

**Comment by ahmedmolawale** [comment]
I'm on it
